build.gradle

```diff
@@ -83,14 +150,6 @@ artifacts {
     archives sourcesJar
 }
 
-task scriptingJavadocs(type: Javadoc) {
```

Comment:
I dont know exactly what this task does, McHorse can tell better, but I am 99.9% sure this is not supposed to be removed.